webots_ros2_tutorials/webots_ros2_tutorials/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 1996-2020 Soft_illusion.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Do we really need this somewhat empty file?

Reply:
![Screenshot from 2020-11-02 22-13-34](https://user-images.githubusercontent.com/16682619/97949315-d43c9600-1d58-11eb-8fb5-c13a0117d2e8.png)

Enble to build the project of we don't have init file.